### PR TITLE
[PLG-376/375] onboarding guide refactor

### PR DESCRIPTION
**Pull request by avivenzio-bitrise**
https://bitrise.atlassian.net/browse/PLG-376
https://bitrise.atlassian.net/browse/PLG-375

- Added info icon back to guide
- Refactored guide to use bitkit accordion
- Learn more link spacing increased & size reduced
- Improve content spacing
- Hide guide on mobile screen size

### Screenshots
Screen Shot 2022-10-17 at 5 41 31 PM

### CODE REVIEW

File: source/javascripts/components/GuidedOnboarding/GuidedOnboardingHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Box, Icon, Text } from "@bitrise/bitkit";
+
+export const GuidedOnboardingHeader = (): JSX.Element => (
+    <Box display="flex" padding="0 18px">
+      <Icon name="Info" />{" "}
+      <Text size="3" fontWeight="bold" paddingLeft="10px">
```
Comment (roland-spekker-bitrise) [inline source/javascripts/components/GuidedOnboarding/GuidedOnboardingHeader.tsx:6]: Isn't size 3 the default for `Text`?

File: source/javascripts/components/GuidedOnboarding/GuidedOnboardingAccordion.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        onToggleOpen(newExpanded.length > 0)
+      }
+    }, [open]);
+  
+    return (
+      <Accordion
+        colorScheme="orange"
+        variant="flat"
+        color="orange.10"
+        index={expandedAccordionIndexArray}
```
[the remark this answers is not in the document]
Comment (avivenzio-bitrise) [reply source/javascripts/components/GuidedOnboarding/GuidedOnboardingAccordion.tsx:37]: Unfortunately we need both, the `useEffect` is there to handle opening the accordion after automatically after the product tour has ended. after this point in time the user will interact with local state.

File: source/javascripts/components/GuidedOnboarding/GuidedOnboardingHeader.tsx (hunk 1 above)
Comment (avivenzio-bitrise) [reply source/javascripts/components/GuidedOnboarding/GuidedOnboardingHeader.tsx]: fixed

File: source/javascripts/components/GuidedOnboarding/GuidedOnboardingAccordion.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (avivenzio-bitrise) [reply source/javascripts/components/GuidedOnboarding/GuidedOnboardingAccordion.tsx:37]: we had a similar implementation in the old `Collapse` component
https://github.com/bitrise-io/bitrise-workflow-editor/pull/707/files#diff-09bd8053e39f57de10fa12688af70e0cfebdd24bdd74f0487b61ba6d48f5b464L22-L27

**Review by roland-spekker-bitrise — APPROVED** [review]

**Pull request merged**